### ISSUE TITLE
Mod doesn't work (1.12.2.)

### ISSUE THREAD

**Issue by nathan-raven** [Exopandora/ShoulderSurfing#49]
I just can't use the mod's features in my modpack. I tried to rebind the keys, but still nothing. I can't move the camera or swap shoulders.

The modpack contains the following mods:

- Controlling-3.0.10
- DynamicSurroundings-1.12.2-3.6.1.0
- MoBends_1.12.2-1.0.0-beta-20.06.20
- music_player-1.12.2-2.1.1.125
- OptiFine_1.12.2_HD_U_G5
- OreLib-1.12.2-3.6.0.1
- ShoulderSurfing-1.12.2-1.15.1
- u_team_core-1.12.2-2.2.5.147
- Wynntils-1.9.1_647-MC1.12.2

Basically, it's Asher's Wynncraft Modpack, I just expanded it a little bit.

**Comment by Exopandora** [comment]
Have you done the suggested [troubleshooting ](https://github.com/Exopandora/ShoulderSurfing#troubleshooting) steps?

**Comment by nathan-raven** [comment]
Yes, I did. Nevermind now, because I fixed the problem by reseting key bindings. I'm currently testing out the mod. I still rebinded some keys, but I didn't mess with the mod's hot keys. Let's see if it works.

**Comment by nathan-raven** [comment]
It works. Sorry for bothering you!

**Comment by Exopandora** [comment]
Alright, no problem!
